src/System Application/App/Language/src/SupportedLanguages.Page.al

```diff
@@ -0,0 +1,30 @@
+namespace System.Globalization;
+
+/// <summary>
+/// Page that shows the list of supported languages which is enabled for this environment. If nothing is specified, then the user will be able to use all available languages.
+/// </summary>
+page 50100 "Supported Languages"
+{
+    Caption = 'Supported Languages';

```

Comment:
I would call it "Allowed Languages" to avoid confusion about is a language is "supported". I get the idea, but we should make it clear in the UI that this is the company/admin who decides what is "allowed" in this environment. "Supported" could maybe confuse some admins/users who go into this page and see it blank (until filled), thinking they are missing all languages, or that none are supported. Goes for the table name too.

Reply:
Not a problem. The current naming was chosen based on Microsoft calls the setting on NST Server for OnPrem installations

Reply:
Got it. Please go for "Allowed Languages" for both table and page name. + @darjoo FYI